README.md

```diff
@@ -13,13 +13,11 @@ submodule under `./asv`
 ## Setup
 
 ```bash
-conda create -n numba-benchmark python=3.6
+conda create -c conda-forge -n numba-benchmark python=3.8 asv
```

Comment:
should we be using the conda-forge build of python?

Reply:
alternative to use conda-forge only for asv
`conda create -n numba-benchmark python=3.8  conda-forge::asv`

Reply:
the `conda-forge::asv` form would be my vote, since it adds `conda-forge` to the solver as lower priority that `defaults`.  Putting just `-c conda-forge` will give it higher priority for all packages.

Reply:
Installing `asv` will also pull in the following:

```
  asv                conda-forge/osx-64::asv-0.4.2-py38hc84c608_0
  python_abi         conda-forge/osx-64::python_abi-3.8-1_cp38
  six                conda-forge/noarch::six-1.15.0-pyh9f0ad1d_0
```

Reply:
`python_abi` is conda-forge's metapackage for abi.
`six` is fine.
i'm just worry that the environment starting with conda-forge's `python` will get mostly replaced when other dependency are installed, making the env less predictable

Reply:
Thanks for the feedback on this, `conda-forge::asv` is what I went for.